### PR TITLE
[#156536597] Fix back button action

### PR DESCRIPTION
**Pull request by asce1062**
#### What does this PR do?

Implements functionality to handle when the back button is pressed on the dashboard.

#### Description of Task to be completed?

- Pressing the back button should display a toast `Press BACK BUTTON again to exit.`
- Pressing the back button again within 2 seconds should exit the app and move it to the background.
- Else remain on the dashboard.

#### How should this be manually tested?

Build the app.

#### What are the relevant pivotal tracker stories?

[#156536597](https://www.pivotaltracker.com/story/show/156536597)

### CODE REVIEW

**Comment by muhallan** [comment]
Please rebase. There are some files that were removed from `develop` but are still in your branch.

File: app/src/main/java/com/andela/art/securitydashboard/SecurityDashboardActivity.java [hunk 1]
```diff
@@ -41,4 +44,25 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+    @Override
+    public void onBackPressed() {
+        if (backButtonToExitPressedTwice) {
+            super.onBackPressed();
+            finish();
+            moveTaskToBack(true);
+        }
+
+        this.backButtonToExitPressedTwice = true;
+        Toast.makeText(this, "Press BACK BUTTON again to exit.",
```
Comment (muhallan) [inline app/src/main/java/com/andela/art/securitydashboard/SecurityDashboardActivity.java]: This `Toast` is displayed twice, even after the second back click, yet it's not needed the second time.

**Comment by asce1062** [comment]
@muhallan Rebased.

File: app/src/main/java/com/andela/art/securitydashboard/SecurityDashboardActivity.java (hunk 1 above)
Comment (asce1062) [reply app/src/main/java/com/andela/art/securitydashboard/SecurityDashboardActivity.java]: Nice catch! On it.

Comment (suada-haji) [inline app/src/main/java/com/andela/art/securitydashboard/SecurityDashboardActivity.java]: Change the Toast message wordings to `Press again to exit.`

Comment (asce1062) [reply app/src/main/java/com/andela/art/securitydashboard/SecurityDashboardActivity.java]: Done.

Comment (asce1062) [reply app/src/main/java/com/andela/art/securitydashboard/SecurityDashboardActivity.java]: Done.

**Review by suada-haji — APPROVED** [review]

**Pull request merged**